<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpAndAway</class>
 <widget class="QMainWindow" name="UpAndAway">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpAndAway</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxKunde">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>691</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Kund*in</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Kunden-ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditKundenID">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditKundenName">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Reisen</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTableWidget" name="tableWidgetKunde">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Reise-ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Beginn der Reise</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ende der Reise</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxReise">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>691</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Reise</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Reise-ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditReiseID">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Buchungen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="tableWidgetReise">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Buchungstyp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Start</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ende</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Preis</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>11</y>
      <width>361</width>
      <height>625</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="Flugbuchung">
        <attribute name="title">
         <string>Flugbuchung</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Buchungs-ID</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditId">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Ankunftsdatum</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateEdit" name="dateEditAnkunft">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Abreisedatum</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDateEdit" name="dateEditAbreise">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Preis</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxPreis">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Abflugort</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEditAbflugort_2">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEditAbflugort">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Ankunftsort</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="lineEditAnkunftsort_2">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="lineEditAnkunftsort">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Airline</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLineEdit" name="lineEditAirline">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Flugklasse</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLineEdit" name="lineEditFlugklasse">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFlugbuchung">
             <property name="text">
              <string>Karte</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Mietwagenreservierung">
        <attribute name="title">
         <string>Mietwagenreservierung</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Buchungs-ID</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditId_2">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Ankunftsdatum</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateEdit" name="dateEditAnkunft_2">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Abreisedatum</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDateEdit" name="dateEditAbreise_2">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Preis</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxPreis_2">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Abholort</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEditAbholort">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Rueckgabeort</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEditRueckgabeort">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Firma</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="lineEditFirma">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Fahrzeugklasse</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="lineEditFahrzeugklasse">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMietwagenreservierung">
             <property name="text">
              <string>Karte</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Hotelreservierung">
        <attribute name="title">
         <string>Hotelreservierung</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Buchungs-ID</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditId_3">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Ankunftsdatum</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateEdit" name="dateEditAnkunft_3">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Abreisedatum</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDateEdit" name="dateEditAbreise_3">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Preis</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxPreis_3">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Hotel</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEditHotel">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Stadt</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEditStadt">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Zimmer</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="lineEditZimmer">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonHotelreservierung">
             <property name="text">
              <string>Karte</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSuche">
       <property name="title">
        <string>Suche</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Kunden-ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxSuche"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSuche">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionEinlesen"/>
    <addaction name="actionSpeichern"/>
   </widget>
   <widget class="QMenu" name="menuNeu">
    <property name="title">
     <string>Hinzufuegen</string>
    </property>
    <addaction name="actionKunde"/>
    <addaction name="actionBuchung"/>
   </widget>
   <widget class="QMenu" name="menuAnzeigen">
    <property name="title">
     <string>Anzeigen</string>
    </property>
    <addaction name="actionChecks"/>
    <addaction name="actionABC_Analyse"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuNeu"/>
   <addaction name="menuAnzeigen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEinlesen">
   <property name="text">
    <string>Einlesen</string>
   </property>
  </action>
  <action name="actionKunde">
   <property name="text">
    <string>Kunde</string>
   </property>
  </action>
  <action name="actionBuchung">
   <property name="text">
    <string>Buchung</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionChecks">
   <property name="text">
    <string>Checks</string>
   </property>
  </action>
  <action name="actionABC_Analyse">
   <property name="text">
    <string>ABC-Analyse</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
